soc/nxp/imxrt/imxrt118x/m33/mpu_regions.c

```diff
@@ -7,13 +7,13 @@
 #include <zephyr/devicetree.h>
 #include <zephyr/arch/arm/cortex_m/arm_mpu_mem_cfg.h>
 
-#define REGION_HYPERRAM_BASE_ADDRESS   0x04000000
-#define REGION_HYPERRAM_SIZE           0x04000000
-#define REGION_DTCM_BASE_ADDRESS       0x20000000
+#define REGION_HYPERRAM_BASE_ADDRESS   0x14000000
+#define REGION_HYPERRAM_SIZE           0x00800000
```

Comment:
```suggestion
#define REGION_HYPERRAM_SIZE           0x04000000
```
Why did you reduce the FlexSPI2 region size?  The FlexSPI2 memory map is from 0x1400_0000 to 0x17FF_FFFF.  Larger memories than the HyperRAM on our EVK could be connected to FlexSPI2.

Reply:
yes, you are right. but I think should be REGION_FLEXSPI2_SIZE ?

Reply:
Agreed, thank you